### ISSUE TITLE
Spinner styling issue when dev_mode set to false

### ISSUE THREAD

**Issue by yoroshiku24** [reduxframework/redux-framework#2532]
1. Redux version: 3.5.6.2
2. WordPress version: 4.2.4
3. dev_mode: false
4. Field code (actually it's from the default sample-config.php):
    ```
    Redux::setSection( $opt_name, array(
        'title'      => __( 'Spinner', 'redux-framework-demo' ),
        'id'         => 'slider_spinner-spinner',
        'desc'       => __( 'For full documentation on this field, visit: ', 'redux-framework-demo' ) . '<a href="//docs.reduxframework.com/core/fields/spinner/" target="_blank">docs.reduxframework.com/core/fields/spinner/</a>',
        'subsection' => true,
        'fields'     => array(
            array(
                'id'      => 'opt-spinner',
                'type'    => 'spinner',
                'title'   => __( 'JQuery UI Spinner Example 1', 'redux-framework-demo' ),
                'desc'    => __( 'JQuery UI spinner description. Min:20, max: 100, step:20, default value: 40', 'redux-framework-demo' ),
                'default' => '40',
                'min'     => '20',
                'step'    => '20',
                'max'     => '100',
            ),
        )
    ) );

    ```
5.  I'm using a theme I crafted myself.
6. Redux is embedded.
7. No javascript error in console.

Issue description:
In dev_mode, the styling of spinner field is normal. When set dev_mode to false, the styling of spinner field becomes unusual, though it still remains functional.
Here's the screenshot: http://i.imgur.com/t2hC3mQ.png

Thanks in advance!

**Comment by kprovance** [comment]
We know.  It's kind of the backburner considering our work load.  If you're up for it, you're free to fix it and submit a pull request.  Otherwise, it may be a little while.

**Comment by kprovance** [comment]
Wait, I see the issue just by looking at it.  Gimme a minute, maybe I can churn this out after all.

**Comment by kprovance** [comment]
Fixed in v3.5.6.4.  :)

**Comment by yoroshiku24** [comment]
Wow, that was extremely fast! Thank you very much for your response :)
